app/javascript/article-form/components/__tests__/Form.test.jsx

```diff
@@ -6,13 +6,29 @@ import { Form } from '../Form';
 let bodyMarkdown;
 let mainImage;
 
+// Axe flags an error for the multi-line combobox we use for Autosuggest, since a combobox should be a single line input.
+// This is a known issue documented on #13044, and these custom rules only apply to the two tests referencing them below.
```

Comment:
```suggestion
// This is a known issue documented on https://github.com/forem/forem/pull/13044, and these custom rules only apply to the two tests referencing them below.
```
Might be nice to link to the PR directly -- the first time I read this, I didn't fully get that the comment was referencing a PR 😆